docs/cheaha/tutorial/pytorch_tensorflow.md

```diff
@@ -108,25 +104,25 @@ print(torch.cuda.get_device_name(x))
 1. Create a new environment that is compatible with supported tensorflow versions, use the below command to do this. For this tutorial we will use Python 3.11.
 
     ```bash
-
     conda create -n tensorflow python=3.11
-
     ```
 
-1. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version use the below command.
+1. Activate the conda environment created in Step 1.
 
     ```bash
+    conda activate tensorflow
+    ```
 
-    pip install --upgrade pip
+1. The TensorFlow CPU and GPU versions requires pip to be up-to-date, to install and upgrade pip to the latest version, within your conda environment use the below command. See our [section](../../cheaha/open_ondemand/ood_jupyter.md#pip-installs-packages-outside-of-environment), on why installing pip within a conda environment is advised.
```

Comment:
I think the first sentence could be two sentences. The first comma would be a good stopping point.